src/Twilio/Rest/Client.php

```diff
@@ -231,7 +235,8 @@ public function request(string $method, string $uri, array $params = [], array $
             $headers,
             $username,
             $password,
-            $timeout
+            $timeout,
+            $logLevel
```

Comment:
Why not just do all the logging here rather than rely on the client implementation to do the logging?